CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+4.60.3
+==========
+### Bug Fixes
+Updated Android webview attribute.
```

Comment:
Is all this complexity worth it? The MSI bug fix related to the CVE is very targetted. 

How about you smoke test 4.60.4 with a public client desktop app and with a UWP app (we don't have coverage anyway) and call it a day?